app/_gateway_entities/key-set.md

```diff
@@ -30,5 +25,20 @@ schema:
 
 ---
 
-@todo
-You can assign one or many keys to a JSON Web Key Set. This can be useful to logically group multiple keys to use for a specific application or service. Key Sets are also the preferred way to expose keys to plugins because they tell the plugin where to look for keys or have a scoping mechanism to restrict plugins to just some keys.
\ No newline at end of file
+## What is a Key Set?
+
+A Key Set is a collection of {{site.base_gateway}} [Keys](/gateway/entities/key).
+
+You can assign one or many Keys to a Key Set. This can be useful to logically group multiple Keys to use for a specific application or service. Key Sets allow you to give a plugin access to a specific list of Keys.
+
```

Comment:
Can you add the list of supported plugins from the original doc here as well? https://docs.konghq.com/gateway/latest/reference/key-management/#key-sets-endpoint 

You could probably add them inline here + in the page metadata as related links. From the source doc, I can't really tell if Keys should also have these links - probably?

Reply:
Ah yes I forgot I needed to ask about this, I think that list may be wrong or out of date, because I only found a key set config parameter on two of these.

For keys I _think_ there's no specific config in the plugins, they just have access to keys by default, but on certain plugins you can specify a key set to restrict access.

Reply:
After looking into it a bit more I think the original doc is about JSON Web Key Sets in general, but not the Key Set entity specifically. Some plugins, like the OIDC one, have their own endpoint to manage JWKS, but they're not necessarily compatible with the Key Set entity. From what I can tell, the only plugins that can use the Key Set entity are JWT Signer, ACME, and JWE Decrypt, so I'll add these.